stdlib/universe/table_fns.go

```diff
@@ -79,21 +79,22 @@ func tableFindCall(ctx context.Context, args values.Object) (values.Value, error
 		}
 	}
 
+	if !execdeps.HaveExecutionDependencies(ctx) {
+		return nil, errors.New(codes.Invalid, "do not have an execution context for tableFind, if using the repl, try executing this code on the server using the InfluxDB API")
```

Comment:
This message feels concerning to me. "If your preferred usage doesn't work, try this other usage." We've learned that there are a lot of people using the REPL, and that that's their preferred way to work with flux. We should absolutely do what we can to make the REPL a first-class flux experience. It looks like the support in the REPL is in this patch, so I'm trying to understand are the conditions whereby this error can occur.

Reply:
Oh the patch is actually meant to fix this case, as it supplies the execution dependencies (last point in the PR description) You're right it is a horrible error message. It's the one I wrote when I first made a stab at this and didn't understand the issues, so I just told the user to not use the repl :)

That path used to cause a panic, so it should probably go back to causing a panic. Or actually probably an internal error would be better, as we should always supply the necessary things for tableFind and friends to work.

https://github.com/influxdata/flux/pull/2558

Reply:
Right, the patch itself didn't seem to align with the error message. I think, given your reply, that a panic might be the most appropriate thing; that state should be impossible _because_ of the changes in the patch. It's essentially an assertion on the code.

Reply:
This is what I don't like about dependency injection, it leaves no room to specify that certain dependencies are mandatory. A context class with a new function with a required argument does so much more to communicate what the dependencies are.